src/express-middleware.js

```diff
@@ -96,9 +115,14 @@ export function createExpressMiddleware(adapter, middlewareOptions = {}) {
     if (adapter.waitForResponse) {
       emitArguments.push(sendResponse(res));
     }
+
     try {
+      debug('emitting event -  type: %s, arguments: %o', req.body.event.type, emitArguments);
       adapter.emit(req.body.event.type, ...emitArguments);
     } catch (error) {
+      // TODO: these errors will never have codes, but otherwise this message should look like the
```

Comment:
this looks strange, but i intend to clean it up in a fix for #18. i just wanted to leave that note there so that when i come back, i rem that there's an opportunity to refactor this line.